src/components/RuntimeManager.js

```diff
@@ -199,10 +184,16 @@ export default class RuntimeManager extends PureComponent {
       })
       errorNotifiedApps.update(Utils.append(galaxyApp.appName))
     }
-  }
+  }, [runtimes, apps]) // eslint-disable-line react-hooks/exhaustive-deps
+
+  return null
```

Comment:
Not sure if there is a better way of defining a pure component in functional react. It errors out if I remove the null return and tells me to put it back. Weird that it doesn't just figure out I don't want anything rendered if I omit the return...

Reply:
Yes, I believe it is required.